sjb/hack/determine_install_upgrade_version.py

```diff
@@ -116,12 +116,9 @@ def sort_pkgs(available_pkgs):
 
 	yb = yum.YumBase()
 	# Have to redirect redirect the yum configuration to /dev/null cause we dont want to have any additional output
-	# that configuration prints. eg: `Loaded plugins: amazon-id, rhui-lb`  
+	# that configuration prints. eg: `Loaded plugins: amazon-id, rhui-lb`
 	old_stdout = sys.stdout
 	sys.stdout = open(os.devnull, 'w')
-	# Need make sure that all the packages are listed, cause if excluders are enabled for some reason,
-	# listing of origin or docker pkgs will be disabled, based on the enabled excluder.
-	yb.conf.disable_excludes = ["all"]
```

Comment:
@jhadvig I'm pretty much 100% sure that this was only necessary when the excluders were all sorts of broken. Both of the times we run this script before we run any ansible installation, so we should be fine to not need this now,